### PR TITLE
Scraper Backend

### LINKED ISSUE #128: [3pts] Run scraper code from DCT Backend
**Definition of Done - Check as you go!**
- [x] Documented
- [x] Code Reviewed

DCT Backend -> calls scraper -> scraper returns -> DCT backend saves returned values

### PR DESCRIPTION
**Pull request by RafaelPiloto10**
This PR integrates the scraper with the dct backend.

### CODE REVIEW

File: dct_backend/server.py [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
+        dct.save_food_item('French Fries', '605 Kitchen', 'Lunch', '100',['carbs'])
+        dct.save_food_item('Ceasar Salad', 'Spice','Dinner','102',['vegan', 'carbs'])
+
+        dct.save_serve_time('Breakfast', '7am - 10am')
+        dct.save_serve_time('Lunch', '11am - 2pm')
+        dct.save_serve_time('Dinner', '2pm - 5pm')
+    else:
+        for entry in output:
+            dct.save_food_item(*entry)
+        print("Successfully scrapped new data")
```
Comment (brendacano) [inline dct_backend/server.py]: did you mean "scraped"? Or did you actually mean scrapped, as in we discarded the new data?

File: dct_backend/server.py [hunk 2]
```diff
@@ -1,29 +1,48 @@
...
 
-app = FastAPI()
 dct = DCT()
+app = FastAPI()
+
+@repeat_every(seconds=60*60*12) # repeat every 12 hours
```
Comment (brendacano) [inline dct_backend/server.py:41]: this is cool

File: dct_backend/server.py (hunk 1 above)
Comment (RafaelPiloto10) [reply dct_backend/server.py]: I can't english

**Review by brendacano — APPROVED** [review]
It all looks very straightforward and clear. Thanks for fixing that typo

**Comment by Foxworth22** [comment]
#140 has been merged

**Review by Foxworth22 — APPROVED** [review]
This all looks good! Does this complete #128?

**Comment by RafaelPiloto10** [comment]
Yes :D

**Pull request merged**